docs/components/task-list.md

```diff
@@ -74,6 +74,8 @@ Add the 'Task list' component to a block list, and add 'Task' components to the
 
 Adding formatting (for example, bold text) in the task name is not currently supported in Umbraco.
 
-The status can be set in the settings of each 'Task', but usually you would [override the status property](/docs/umbraco/override-property-values.md) in code. Setting a status other than those in the `TaskListTaskStatus` enum is not currently supported.
+The status text content can be set in the content of each 'Task' using the 'Status text' property. If this value is not set then the dictionary value assigned to the `TaskListStatusEnum` will be used. If no dictionary value is available then the text of the enum will be used. Usually you would override [this content property](/docs/umbraco/override-property-values.md) in code.
```

Comment:
You're talking about the dictionary value before the dictionary value is explained. Also, because the need to use 'status text' should be the exception rather than the rule, it makes sense to explain it last. 

You've already got the explanation of the default text and dictionary below. I would remove the paragraph you have, and add a simpler paragraph lower down after the bit about the dictionary. Something like "Use the 'Status text' property to override the status text for an individual task when using the dictionary entry is not flexible enough for your application."